lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -73,7 +73,7 @@ class GoogleAdsClient
 
         DEFAULT_CONFIG_FILENAME = 'google_ads_config.rb'
 
-        attr_reader :logger
+        attr_accessor :logger
```

Comment:
Can you please undo the changes for attr_reader vs. attr_accessor with the logger? I specifically separated them out partially for the comment to have a place to exist but also because there's a chance that more needs to be done on setting the logger in the future than simple assignment.

Reply:
ok sure